cluster/minikube.go

```diff
@@ -77,7 +77,7 @@ func (cluster *minikubeCluster) Delete() error {
 
 // DockerEnv returns the environment variables necessary to connect to the private docker repository in the kubernetes cluster
 func (cluster *minikubeCluster) DockerEnv() ([]string, error) {
-	statusCmd := exec.Command("minikube", "docker-env", "-p", cluster.Name)
+	statusCmd := exec.Command("minikube", "docker-env", "-p", cluster.Name, "--shell", "bash")
```

Comment:
Had a quick look at the file and it seems that we are not checking errors for some cmd executions - not sure if its expected.

Reply:
Other: Also the internal/build/utils.go includes some utils for cmds. I'm not sure if its the best place though

Reply:
@rgeraldes In all cmd executions we check for it's exit status to be 0. Where did you find a cmd without error checks?